src/index.js

```diff
@@ -0,0 +1,48 @@
+import './index.css';
+import { postData, getData } from './modules/getapi.js';
+
+const form = document.getElementById('add-form');
+const container = document.getElementById('scores-container');
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const refreshBtn = document.getElementById('refresh-btn');
+
+const displayScore = (person, personScore) => {
+  const tableRow = document.createElement('tr');
+  tableRow.classList.add('person-1');
+  tableRow.innerHTML = `
+   <td class="title">${person}</td>
+   <span> : </span>
+   <td class="scores">${personScore}</td>`.trim();
+
+  container.appendChild(tableRow);
+};
+
+form.addEventListener('submit', async (e) => {
+  e.preventDefault();
+  const person = name.value;
+  const personScore = score.value;
+  await postData({
+    user: person,
+    score: Number(personScore),
+  });
+
+  const scores = await getData();
+  scores.result.forEach((score) => {
+    displayScore(score.user, score.score);
+  });
+});
+
+document.addEventListener('DOMContentLoaded', async () => {
+  const scores = await getData();
+  scores.result.forEach((score) => {
+    displayScore(score.user, score.score);
+  });
+});
```

Comment:
On this part, it's recommended that the page loads empty and the list of data is shown as soon as one click on the refresh button. 

- [ ] To my observation, you don't need this code here. It would be nice if you could remove it and only show data when the refresh button is clicked as required in the project requirement.

Reply:
thanks for pointing out the fact and thanks for your suggestion. I will change that. ✨